man/pam_systemd.xml

```diff
@@ -159,6 +159,19 @@
           </tgroup>
         </table>
 
+        <para>If no session class is specified via either the PAM module option, nor via the
+        <varname>XDG_SESSION_CLASS</varname> environment variable, the class is automatically chosen, depending on
+        various session parameters, such as the session type (if known), whether the session has a TTY or X11
+        display, and the user disposition. Note that various tools allow setting the session class for newly
+        allocated PAM sessions explicitly and in a fine-grained fashion, by means of the
+        <varname>XDG_SESSION_CLASS</varname> environment variable. For example, classic UNIX cronjobs understand
+        environment variable assignments (see
+        <citerefentry project='man-pages'><refentrytitle>crontab</refentrytitle><manvolnum>5</manvolnum></citerefentry>),
+        which maybe used to choose between the <constant>background</constant> and
+        <constant>background-light</constant> session class individually per cronjob, or
+        "run0 --setenv=<varname>XDG_SESSION_CLASS</varname>" may be used to choose between <constant>user</constant>
```

Comment:
enclose the full command in `<command>` and to provide a full illustrative example, set the env var to "user-light".